test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs

```diff
@@ -62,19 +62,19 @@ internal static void RunTest (MethodDeclarationSyntax m, List<AttributeSyntax> a
 			foreach (var attr in attrs) {
 				switch (attr.Name.ToString ()) {
 				case "ExpectedWarning":
-					var expectedMessageContains = attr.ArgumentList!.Arguments.Select (m => GetStringFromExpr (m.Expression)).ToList ();
-					if (!expectedMessageContains[0].StartsWith ("IL"))
+					var expectedMessagesContains = attr.ArgumentList!.Arguments.Select (m => GetStringsFromExpr (m.Expression)).ToList ();
+					var expectedWarningCode = expectedMessagesContains[0];
+					if (!expectedWarningCode[0].Contains ("IL"))
 						break;
-					expectedMessageContains.RemoveAt (0);
+					var expectedMessages = expectedMessagesContains[1];
```

Comment:
It looks like this no longer handles `params` arrays properly, namely it only looks at the second argument, instead of all the arguments.